orchestration/pom.xml

```
@@ -140,72 +144,39 @@
       <build>
         <plugins>
           <plugin>
-            <groupId>org.openapitools</groupId>
+            <groupId>com.sap.cloud.sdk.datamodel</groupId>
             <artifactId>openapi-generator-maven-plugin</artifactId>
+            <configuration>
+              <skip>true</skip>
+              <!-- TODO: remove this once Cloud SDK 5.15.0 is released -->
+              <outputDirectory>${project.basedir}/src/main/java</outputDirectory>
+              <apiMaturity>released</apiMaturity>
```

```suggestion
              <apiMaturity>beta</apiMaturity>
```

we can't guarantee API stability